src/App.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React, { useEffect } from 'react';
 import { withRouter, Switch, Route } from 'react-router-dom';
-import { getRegistry } from '@red-hat-insights/insights-frontend-components';
+import { getRegistry } from '@red-hat-insights/insights-frontend-components/Utilities/Registry';
```

Comment:
I know that it's not really focus of this PR but can you use `@redhat-cloud-services/frontend-components-utilities`? Or we can create issue to migrate to the new packages if you feel that it would be too much to handle in this PR.

Reply:
I'll do it in another PR